open-api/swagger.yaml

```diff
@@ -1929,6 +1937,40 @@ paths:
       responses:
         204:
           description: 'returns null'
+  /v2/query_diffs/{queryId}:
```

Comment:
Wouldn't `/v2/queries/{queryId}/updates` make more sense here? They refer to the same resource `/v2/queries/{queryId}`.

Reply:
@gijskant True, so as v2/query_diffs/scan. Do you think it should be changed to /v2/queries/scan? Would it be better to remove UserQueryDiffController and move all its methods to UserQueryController then?